pages/homePage/homePage.css

```diff
@@ -0,0 +1,197 @@
+@import "/styles.css";
+
+.page-wrap {
+
+    grid-template-areas:
+        'p_nav p_nav p_nav p_nav p_nav p_nav'
+        'p_content p_content p_content p_content p_content p_content'
+        'p_content p_content p_content p_content p_content p_content'
+    ;
+}
+
+/* Header */
+.page-header {
+    grid-area: p_header;
+}
+
+.page-sidebar {
+    display: none;
+}
+
+.page-footer {
+    display: none;
+}
+
+.main-content {
+    width: 100%;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+}
+
+.main-content__header {
+    height: 100%;
+}
+
+.promo-catagory-holder {
+    display: flex;
+    width: 100%;
+    height: 50%;
+    gap: 2rem;
+    padding: 0.8rem
```

Comment:
this value can be in pixels.

Reply:
My functionality as of now require it to be rem.